src/main/java/org/jenkinsci/plugins/codesonar/models/CodeSonarAlertFrequencies.java

```diff
@@ -0,0 +1,70 @@
+package org.jenkinsci.plugins.codesonar.models;
+
+/**
+ * @author aseno
+ *
+ */
+public class CodeSonarAlertFrequencies {
+    private int red;
+    private int yellow;
+    private int blue;
+    private int green;
+    
+    public CodeSonarAlertFrequencies() {
+        red = 0;
+        yellow = 0;
+        blue = 0;
+        green = 0;
+    }
+    
+    public int getRed() {
+        return red;
+    }
```

Comment:
I think these getters and settings should say that they are returning a *count*.  E.g. `getRedCount()` - or if you use my previous suggestion, `getAlertCount(AlertLevel.Red)`.